components/collapse/nz-collapse-panel.component.ts

```diff
@@ -61,13 +76,23 @@ export class NzCollapsePanelComponent implements OnInit, OnDestroy {
     this.cdr.markForCheck();
   }
 
-  constructor(private cdr: ChangeDetectorRef, @Host() private nzCollapseComponent: NzCollapseComponent) {
+  private addIconClass(): void {
```

Comment:
I guess these codes are redundant if you change the implementation.

Reply:
done